core/bin/external_node/src/config/mod.rs

```diff
@@ -146,6 +147,10 @@ impl RemoteENConfig {
             .get_main_contract()
             .rpc_context("get_main_contract")
             .await?;
+        let timestamp_asserter_address = client
```

Comment:
Is there a risk that the main node won't have this method implemented (e.g., on the mainnet)? If so, it makes sense to handle `MethodNotFound` errors as it's done below.

Reply:
Agreed and fixed